src/qibotf/custom_operators/python/ops/qibo_tf_custom_operators.py

```diff
@@ -4,11 +4,11 @@
 if tf.__version__ < __target_tf_version__:  # pragma: no cover
     raise RuntimeError(
         f"qibotf {__version__} requires TensorFlow {__target_tf_version__}."
-        "Please upgrade TensorFlow or downgrade qibotf.")
+        " Please upgrade TensorFlow or downgrade qibotf.")
 elif tf.__version__ > __target_tf_version__:  # pragma: no cover
     raise RuntimeError(
         f"qibotf {__version__} requires TensorFlow {__target_tf_version__}."
-        "Please upgrade qibotf or downgrade TensorFlow.")
+        " Please upgrade qibotf or downgrade TensorFlow.")
```

Comment:
Aren't these checks equivalent to a single `if tf.__version__ != __target_tf_version__` check? 

Also regarding the error message, I think there is also the option to just rebuild the custom operators without upgrading/downgrading any of the libraries. However this may be confusing for the average user so it is okay to leave it as it is.

Reply:
Yes, indeed, should be equivalent. The if/else is here just to explain to the average user how to solve the incompatibility. 
But, I agree that a simple != should make it, e.g. we could say `qibotf and tensorflow do not match, please recompile qibotf or check the documentation`. What do you think?

Reply:
> But, I agree that a simple != should make it, e.g. we could say `qibotf and tensorflow do not match, please recompile qibotf or check the documentation`. What do you think?

This sounds good and is a bit more general than the current. I would just add "qibotf and tensorflow **versions** do not match". I am fine with either message, but perhaps referring to the docs is the best idea.